features/support/stub_survey_monkey_requests.rb

```diff
@@ -0,0 +1,5 @@
+require 'webmock/cucumber'
+
+Before('@stub_survey_monkey_success') do
+  @called_zendesk = stub_request(:post, %r{\Ahttps://api.eu.surveymonkey.com/v3/.*} ).and_return(status: 201, body: { id: 123 }.to_json )
```

Comment:
💯

Reply:
is the response required/used, plus the name would be wrong if it is needed
```suggestion
  stub_request(:post, %r{\Ahttps://api.eu.surveymonkey.com/v3/.*} )
    .and_return(status: 201, body: { id: 123 }.to_json )
```

We should  also have a negative/error test and stub

Reply:
I have added a negative test.